configure.ac

```diff
@@ -1786,6 +1786,13 @@
     ;;
     esac
 
+    AC_CHECK_LIB(unwind,unw_backtrace,,LIBUNW="no")
```

Comment:
for this backport I would not want it to use this by default, but instead only compile it in if `--enable-...` is specified.